src/pyOpenMS/pxds/PeakFileOptions.pxd

```diff
@@ -7,72 +7,73 @@ from MSNumpressCoder cimport *
+        void setSkipXMLChecks(bool only) nogil except + # wrap-doc:Sets whether to skip some XML checks and be fast instead
+        bool getSkipXMLChecks() nogil except + # wrap-doc:Returns whether to skip some XML checks and be fast instead
 
-        bool getWriteIndex() nogil except +
-        void setWriteIndex(bool write_index) nogil except +
+        bool getWriteIndex() nogil except + # wrap-doc:Returns whether to write an index at the end of the file (e.g. indexedmzML file format)
+        void setWriteIndex(bool write_index) nogil except + # wrap-doc:Returns whether to write an index at the end of the file (e.g. indexedmzML file format)
 
-        NumpressConfig getNumpressConfigurationMassTime() nogil except +
-        void setNumpressConfigurationMassTime(NumpressConfig config) nogil except +
+        NumpressConfig getNumpressConfigurationMassTime() nogil except + # wrap-doc:Sets numpress configuration options for m/z or rt dimension
+        void setNumpressConfigurationMassTime(NumpressConfig config) nogil except + # wrap-doc:Returns numpress configuration options for m/z or rt dimension
 
-        NumpressConfig getNumpressConfigurationIntensity() nogil except +
-        void setNumpressConfigurationIntensity(NumpressConfig config) nogil except +
+        NumpressConfig getNumpressConfigurationIntensity() nogil except + # wrap-doc:Sets numpress configuration options for intensity dimension
+        void setNumpressConfigurationIntensity(NumpressConfig config) nogil except + # wrap-doc:Returns numpress configuration options for intensity dimension
 
-        NumpressConfig getNumpressConfigurationFloatDataArray() nogil except +
-        void setNumpressConfigurationFloatDataArray(NumpressConfig config) nogil except +
+        NumpressConfig getNumpressConfigurationFloatDataArray() nogil except + # wrap-doc:Sets numpress configuration options for float data arrays
+        void setNumpressConfigurationFloatDataArray(NumpressConfig config) nogil except + # wrap-doc:Returns numpress configuration options for float data arrays
 
-        void setForceMQCompatability(bool forceMQ) nogil except +
-        bool getForceMQCompatability() nogil except +
+        void setForceMQCompatability(bool forceMQ) nogil except + # wrap-doc:[mzXML only!]Returns Whether to write a scan-index and meta data to indicate a Thermo FTMS/ITMS instrument (required to have parameter control in MQ)
+        bool getForceMQCompatability() nogil except + # wrap-doc:[mzXML only!]Returns Whether to write a scan-index and meta data to indicate a Thermo FTMS/ITMS instrument (required to have parameter control in MQ)
 
-        void setForceTPPCompatability(bool forceTPP) nogil except +
-        bool getForceTPPCompatability() nogil except +
+        void setForceTPPCompatability(bool forceTPP) nogil except +[ mzML only!]Returns Whether to skip writing the \<isolationWindow\> tag so that TPP finds the correct precursor m/z
```

Comment:
`# wrap-doc:` missing

Reply:
Thankyou @oliveralka. I have fixed this now.